.github/workflows/create-release.yml

```diff
@@ -144,14 +167,16 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           REPOSITORY: ${{ github.repository }}
         run: |
-          RELEASE_ID=$(./hack/scripts/create_draft_release.sh)
+          ./scripts/shell/create_draft_release.sh
+          RELEASE_ID=$(cat release_id.txt)
+          echo "Release ID: $RELEASE_ID"
           echo "release_id=$RELEASE_ID" >> $GITHUB_OUTPUT
     outputs:
       release_id: ${{ steps.create-draft.outputs.release_id }}
```

Comment:
Why do we output the release id here?

Reply:
If you are talking about `release_id: ${{ steps.create-draft.outputs.release_id }}`, then this is required way to export output from a step so that other steps can use it.

Reply:
Are there other steps using this value? I could not spot any!

Reply:
atm no, but later if we want to auto publish the release, it will be used there.